src/map/status.cpp

```diff
@@ -13331,7 +13327,7 @@ int32 status_change_end( struct block_list* bl, enum sc_type type, int32 tid ){
 			unit_data *ud = unit_bl2ud(bl);
 
 			if (ud)
-				ud->attackabletime = ud->canact_tick = ud->canmove_tick = gettick();
+				ud->attackabletime = ud->canact_tick = ud->canmove_tick = ud->endure_tick = gettick();
```

Comment:
```suggestion
				ud->attackabletime = ud->canact_tick = ud->canmove_tick = gettick();
				ud->endure_tick = gettick();
			}
```

Reply:
Track changes / git blame.
If you want something like this, please define a variable above and asign it to all.